### PR TITLE
Updated ReadMe

### PR DESCRIPTION
**Pull request by KaluBuikem**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Basic principles for operation are:
   servers.
 
 JupyterHub also provides a
-[REST API][]
+[REST API]:https://jupyterhub.readthedocs.io/en/latest/reference/rest-api.html
```
Comment (Mackenzie-OO7) [inline README.md:59]: Hi, I suggest the REST API link be embedded in "REST API".

Comment (KaluBuikem) [reply README.md:59]: Thanks a lot for the suggestion I would implement it immediately

Comment (Mackenzie-OO7) [reply README.md]: > Thanks a lot for the suggestion I would implement it immediately

You're welcome 👍🏾

File: README.md [hunk 2]
```diff
@@ -56,10 +56,10 @@ Basic principles for operation are:
   servers.
 
 JupyterHub also provides a
-[REST API][]
+[REST API](https://jupyterhub.readthedocs.io/en/latest/reference/rest-api.html)
```
Comment (minrk) [inline README.md:59]: Using markdown references this way was deliberate. This is not a change we should make.

**Pull request closed without merging**